CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ContinuousParameter` base class
 - New environment variable `BAYBE_CACHE_DIR` that can customize the disk cache directory
   or turn off disk caching entirely
+- Better human readable `__str__` representation of search spaces
```

Comment:
hey @RimRihana please could you upload one more screenshot of what the new output looks likeafter the changes? Other than that I think its close to approve

Reply:
Yes, sure!

- One example for a hyprid search space:
![image](https://github.com/emdgroup/baybe/assets/118910837/1b155bf6-a21a-482e-a2c6-d09052aaaa6b)

- Another one for a discrete search space:
![image](https://github.com/emdgroup/baybe/assets/118910837/a0f3f86b-3213-4a8d-9ca2-6e4dd6509abb)

Reply:
can we get rid of these line breaks? 
![image](https://github.com/emdgroup/baybe/assets/17951239/ba09547a-4fad-45d5-ac6e-25aeb3d303d1)
in general you can me more sparse on the line breaks to save more space